core/src/main/java/com/scalar/db/transaction/consensuscommit/Coordinator.java

```diff
@@ -28,6 +30,15 @@
 public class Coordinator {
   public static final String NAMESPACE = "coordinator";
   public static final String TABLE = "state";
+  public static final TableMetadata TABLE_METADATA =
+      TableMetadata.newBuilder()
+          .addColumn(Attribute.ID, DataType.TEXT)
+          .addColumn(Attribute.STATE, DataType.INT)
+          .addColumn(Attribute.CREATED_AT, DataType.BIGINT)
+          .addColumn(Attribute.METADATA, DataType.TEXT)
```

Comment:
@feeblefakie Do we need this `METADATA` column for the Coordinator table, correct?

Reply:
We don't use it currently, so we don't need it.
But, I added this for schema flexibility by storing a serialized JSON string or something.

Reply:
Okay, so do we need to remove the `METADATA` related things in another PR?